project.clj

```diff
@@ -25,11 +25,12 @@
                      :url                     "http://www.apache.org/licenses/LICENSE-2.0"}
   :min-lein-version "2.8.1"
   :repositories     [["sonatype-snapshots" {:url "https://oss.sonatype.org/content/groups/public" :snapshots true}]
-                     ["jitpack"            {:url "https://jitpack.io"}]]
+                    ["jitpack" "https://jitpack.io"]]
```

Comment:
Please preserve horizontal alignment in statically initialised data structures (in this case the `:repositories` vector).